pom.xml

```diff
@@ -525,6 +525,23 @@
                 <version>${carbon.identity.version}</version>
             </dependency>
 
+            <!-- User Profile Mgt Tests -->
+            <dependency>
+                <groupId>org.wso2.carbon.identity</groupId>
+                <artifactId>org.wso2.carbon.identity.user.profile.stub</artifactId>
+                <version>${carbon.identity.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.identity</groupId>
+                <artifactId>org.wso2.carbon.idp.mgt.stub</artifactId>
+                <version>${carbon.identity.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.commons</groupId>
+                <artifactId>org.wso2.carbon.user.mgt.stub</artifactId>
+                <version>${carbon.commons.version}</version>
+            </dependency>
+
```

Comment:
I think we need to mark the scope as test for the above dependencies and we can get rid of the comment